src/components/Main/state/URLSerializer.ts

```diff
@@ -60,7 +61,10 @@ export function deserializeScenarioFromURL(initState: State) {
         containment: {
           ...initState.containment,
           current: obj.current.containment,
-          data: { reduction: containmentDataReduction },
+          data: {
+            reduction: containmentDataReduction,
+            numberPoints: initState.containment.data.numberPoints,
+          },
```

Comment:
@brunorzn Does this small edit look ok to you? This was causing a TS error since `numberPoints`was missing.

Reply:
Yes, it loos good! Thanks!